includes/polling/os/eltek.inc.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * eltek.inc.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 Mikael Sipilainen
+ * @author     Mikael Sipilainen <mikael.sipilainen@gmail.com>
+ */
+
+$version   = snmp_get($device, '.1.3.6.1.2.1.1.1.0', '-OQva', 'SNMPv2-MIB');
```

Comment:
You don't need to re-fetch this, just use:
```php
$version = $device['sysDescr'];
```

Reply:
Hopefully updated version looks better now.